libnetwork/Host.cpp

```diff
@@ -60,6 +62,7 @@ using namespace dev::network;
 using namespace dev::eth;
 using namespace dev::crypto;
 
+const std::string Host::split_char = "#";

```

Comment:
really really think this var is unnecessary

Reply:
done, please refer to: https://github.com/FISCO-BCOS/FISCO-BCOS/pull/600/files#diff-df0545cce20878ff6751b98f9348901cR215